snuba/datasets/events.py

```diff
@@ -345,6 +345,26 @@ def column_expr(
         )
         if processed_column:
             # If processed_column is None, this was not a tag/context expression
+
+            # This conversion must not be ported to the errors dataset. We should
```

Comment:
I put this logic in events because:
1) This is a feature we cannot properly support. So it should not pretend to be a generic solution in the tag resolver.
2) I still want the response of a query on the events storage to return the same thing if it is queries on its own from the events dataset and if it is queries through the discover dataset. 


I decided to return `'True'` / `'False'` (instead of `'true'`/`'false'` or `'1'`/`'0'` so that it is compatible with the result we will get with the error dataset and it keeps consistency with a non-promoted version of this context.
But all options are horrible since the actual value provided by the user is lost forever when we normalize the input in the message processor.